README.md

```diff
@@ -117,37 +122,50 @@ Add the Android platform as the target for your mobile app. Then install plug-in
 ```
 $ cd mobile
 $ cordova platform add android
+```
 
-# ensure that everything has been installed correctly
+Ensure that everything has been installed correctly:
+
+```
 $ cordova requirements
+```
+
+Install the plugins required by the application:
 
+```
 $ cordova plugin add cordova-plugin-camera
 $ cordova plugin add cordova-plugin-file-transfer
 ```
 
-### 6. Setup your Android phone
+### 6. Setup your Android device
 
-In order to run the mobile app on your phone, you will need to enable `developer options` and `web debugging`.
+In order to run the application on your Android device, you will need to be prepared to transfer the application's `.apk` file to your device (created in the next step). There are multiple ways for developers to achieve this.
 
-> Note: Please refer to documentation on your specific phone to set these options.
+Android Studio will handle the transfer for you if you tether your Android device to your computer, and enable enable both `developer options` and `web debugging`.
 
-Attach your Android phone to your computer.
+> Note: Please refer to documentation on your specific phone to set these options.
 
-Install [Android File Transfer](https://www.android.com/filetransfer/) on your computer to enable file transfer between your computer and phone.
+For Mac users, [Android File Transfer](https://www.android.com/filetransfer/) will facilitate simple file transfers between your computer and Android device.
 
 ### 7. Build and run the mobile app
 
 ```
 $ cd mobile
 $ cordova build android
+```
+
+An `.apk` file should appear at `mobile/platforms/android/build/outputs/apk/android-debug.apk`, which contains the Android application. You can manually transfer the `.apk` to your device, or if your device is tethered (as described in the previous step), then you can run:
+
+```
 $ cordova run android
 ```
 
-At this point, the app named `Calorie Counter` should come up on your phone. Press the `Capture Image` button to transfer to your phone camera. Take a photo of a food item and press `OK`. Analysis data will then be displayed.
+At this point, the app named `Calorie Counter` should come up on your phone. Use the camera icon to take a photo of a food item, and allow Watson to analyze the image and fetch the calorie results.
 
 # Sample Output
 
-<img src="doc/source/images/output1.jpg" width="250">  <img src="doc/source/images/output2.jpg" width="250">
+![Captured image](doc/source/images/output1.jpg)
```

Comment:
i think the size constraints made things a bit more user friendly TBH

Reply:
Can I just reduce the size of the original images, then? I don't think they need to be 720x1280 for documentation purposes, do they?

Reply:
for sure

Reply:
@dolph @stevemart sample output will be changed since UI has been improved in the last commit.

Reply:
Reduced both image sizes to 360x640 in a subsequent commit, which fit well side-by-side in Github's markdown rendering, but I still need to update the screenshots with the new UI.

Reply:
@dolph Updated the screenshot with new UI in dev5 branch.